### PR TITLE
Added Online Python Tutor

### PR DESCRIPTION
**Pull request by bhavnanisidhant**
I found this tool helpful when I started to learn how to code and I thought that this should be on the list.

### CODE REVIEW

File: docs/intro/learning.rst [hunk 1]
```diff
@@ -30,7 +30,13 @@ If you want a more traditional book, *Python For You and Me* is an excellent
 resource for learning all aspects of the language.
 
     `Python for You and Me <http://pymbook.readthedocs.org/>`_
```
Comment (sigmavirus24) [inline docs/intro/learning.rst]: You need another new-line here.

File: docs/intro/learning.rst [hunk 2]
```diff
@@ -30,7 +30,13 @@ If you want a more traditional book, *Python For You and Me* is an excellent
...
+Online Python Tutor: a basic understanding of how your code will run
```
Comment (sigmavirus24) [inline docs/intro/learning.rst]: Get rid of ": a basic understanding of how your code will run"

File: docs/intro/learning.rst [hunk 3]
```diff
@@ -30,7 +30,13 @@ If you want a more traditional book, *Python For You and Me* is an excellent
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (sigmavirus24) [inline docs/intro/learning.rst]: Make sure this is the same length as "Online Python Tutor" once you've removed everything else.

**Comment by sigmavirus24** [comment]
Please address my comments and then ping me to re-review this.

**Comment by bhavnanisidhant** [comment]
@sigmavirus24 I made the changes.

**Comment by bhavnanisidhant** [comment]
Could we also curate a list of all the meetup groups from around the world?

**Comment by sigmavirus24** [comment]
> Could we also curate a list of all the meetup groups from around the world?

No. That's not the purpose of the guide.

**Pull request merged**